keras/layers/reshaping/up_sampling2d.py

```diff
@@ -140,11 +140,8 @@ def _resize_images(
         Returns:
             A tensor.
         """
-        if data_format == "channels_first":
-            rows, cols = 2, 3
-        elif data_format == "channels_last":
-            rows, cols = 1, 2
-        else:
+        rows, cols = 1, 2
```

Comment:
You can simply remove this line, and hardcode 1 and 2 on line 162.

Reply:
Done